src/metabase/driver.clj

```diff
@@ -43,6 +43,12 @@
      java.time.format.TextStyle/SHORT
      (java.util.Locale/getDefault))))
 
+(defn- long-timezone-name [timezone-id]
+  (let [^java.time.ZoneId zone (if (seq timezone-id)
+                                 (t/zone-id timezone-id)
+                                 (t/zone-id))]
+     zone ))
```

Comment:
This can be done without the `let` wrapping, by putting that type annotation before the parameter list:
```clojure
(defn- long-timezone-name ^java.time.ZoneId [timezone-id]
  (if (seq timezone-id)
    (t/zone-id timezone-id)
    (t/zone-id)))
```

Reply:
Thank you Braden o/